src/voprf.rs

```diff
@@ -585,9 +588,15 @@ struct BatchItems<G: Group, H: BlockInput + Digest> {
     blinded_element: BlindedElement<G, H>,
 }
 
-/// Convenience test functions for [BlindedElement], [EvaluationElement], and [Proof]
-
 impl<G: Group, H: BlockInput + Digest> BlindedElement<G, H> {
+    /// Only used to easier validate allocation
+    fn copy(&self) -> Self {
+        Self {
+            value: self.value,
+            hash: PhantomData,
+        }
+    }
```

Comment:
I introduced this to help search for unnecessary copies. We can't make `BlindedElement` `Copy` because it would prevent `Zeroize` from being implemented, but because it's underlying data is `Copy`, the compiler produces the same assembly output (tested it).